src/config.rs

```diff
@@ -621,4 +620,296 @@ Message w/out In-Reply-To
 
         Ok(())
     }
+
+    enum ChatForTransition {
+        OneToOne,
+        GroupChat,
+        VerifiedGroup,
+    }
+    use ChatForTransition::*;
+
+    // TODO refactoring: Should this really be placed here? But I wouldn't know a better place, either
+    #[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+    async fn test_aeap_transition_0() {
+        check_aeap_transition(OneToOne, false, false).await;
```

Comment:
Yes, we can't call `check_aeap_transition(chat_for_transition=VerifiedGroup, verified=false, ...)` since there is no verified group if `verified=false`